conngen/conngen.cpp

```diff
@@ -165,13 +64,13 @@ cg_connect( ConnectionGeneratorDatum& cg,
     // connect source to target
     while ( cg->next( source, target, NULL ) )
     {
-      if ( kernel().node_manager.is_local_gid( target + target_offset ) )
+      if ( kernel().node_manager.is_local_gid( target_gids[ target ] ) )
```

Comment:
OK, so the function I commented on last time was replaced, but same thing applies here: Why this test?